ProcessLib/HeatTransportBHE/HeatTransportBHEProcessData.h

```diff
@@ -25,12 +26,29 @@ namespace ProcessLib::HeatTransportBHE
 {
 struct HeatTransportBHEProcessData final
 {
+    HeatTransportBHEProcessData(
+        std::unique_ptr<MaterialPropertyLib::MaterialSpatialDistributionMap>&&
+            media_map_,
+        std::vector<BHE::BHETypes>&& vec_BHEs_,
+        bool const& has_network_python_bc_,
+        BHEInflowPythonBoundaryConditionPythonSideInterface* py_bc_object_ =
+            nullptr)
+        : media_map(std::move(media_map_)),
+          _vec_BHE_property(std::move(vec_BHEs_)),
+          has_network_python_bc(has_network_python_bc_),
+          py_bc_object(py_bc_object_)
+    {
+    }
     std::unique_ptr<MaterialPropertyLib::MaterialSpatialDistributionMap>
         media_map;
 
     std::vector<BHE::BHETypes> _vec_BHE_property;
 
     MeshLib::PropertyVector<int> const* _mesh_prop_materialIDs = nullptr;
     std::unordered_map<int, int> _map_materialID_to_BHE_ID{};
+    // if bhe network exists python boundary condition
+    bool has_network_python_bc;
+    //! Python object computing BC values.
+    BHEInflowPythonBoundaryConditionPythonSideInterface* py_bc_object;
```

Comment:
Minor:
Couldn't the two new members be reduced to the pointer?
I'd think if the pointer is null, then there is no python BC making the `has_network_python_bc` unnecessary.

Reply:
It is not needed. Removed.